integration-cli/trust_server.go

```diff
@@ -12,19 +12,23 @@ import (
 	"time"
 
 	"github.com/docker/docker/cliconfig"
+	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/pkg/tlsconfig"
-	"github.com/docker/notary/client"
-	"github.com/docker/notary/passphrase"
-	"github.com/docker/notary/tuf/data"
 	"github.com/go-check/check"
 )
 
 var notaryBinary = "notary-server"
 var notaryClientBinary = "notary"
```

Comment:
as a side discussion: can we rename these? I think notaryBinary should refer to the actual "notary" binary and the other can just be notaryServerBinary as it refers to the "notary-server" binary. WDYT?